Sources/SQLiteNIO/SQLiteData.swift

```diff
@@ -101,9 +101,7 @@ public enum SQLiteData: Equatable, Encodable, CustomStringConvertible {
         case .integer(let value): try container.encode(value)
         case .float(let value): try container.encode(value)
         case .text(let value): try container.encode(value)
-        case .blob(var value):
-            let bytes = value.readBytes(length: value.readableBytes) ?? []
-            try container.encode(bytes)
+        case .blob(let value): try container.encode(Array(value.readableBytesView)) // N.B.: Don't use ByteBuffer's Codable conformance; it encodes as Base64, not raw bytes
```

Comment:
no better idea than turning it into an Array? That's not going to be fun i think ...

Reply:
It's identical to the original code in terms of the Array conversion (`readBytes()` effectively does the exact same thing). To do any better I'd have to either add an ersatz Encodable conformance to ByteBufferView (which gets dicey if NIO ever added their own) or break the API.

It's not that big a deal, though - encoding `SQLiteData` values isn't a standard or expected usage; I'm not even sure why that conformance is there; we don't make use of it at any layer of the stack.